spec/memo_wise_spec.rb

```diff
@@ -203,6 +218,21 @@ def public_memowise_method
       expect(instance.with_keyword_args_counter).to eq(4)
     end
 
+    it "resets memoization for methods for specific keyword arguments" do
+      instance.with_keyword_args(a: 1, b: 2)
+      instance.with_keyword_args(a: 2, b: 3)
+      instance.reset_memo_wise(:with_keyword_args, a: 1, b: 2)
+
+      expect(Array.new(4) { instance.with_keyword_args(a: 1, b: 2) }).
+        to all eq("with_keyword_args: a=1, b=2")
+
+      expect(Array.new(4) { instance.with_keyword_args(a: 2, b: 3) }).
+        to all eq("with_keyword_args: a=2, b=3")
+
+      # This should be executed twice for each set of arguments passed
```

Comment:
(see above -- trying to have the comment draw a line from _we reset 1 out of 2 memoized args_ through _we expect 3 total calls here_ is what I am looking for)